src/p10/p10_edbgEcmdDllScom.C

```diff
@@ -437,7 +436,52 @@ uint32_t p10_dllPutScom(const ecmdChipTarget &i_target, uint64_t i_address,
   struct pdbg_target *addr_base;
   std::string pdbgClassString;
 
-  if (i_target.chipType == "explorer") {
+  if (i_target.chipType == "odyssey") {
+    rc = odyssey_convertCUString_to_pdbgClassString(i_target.chipUnitType,
+                                                    pdbgClassString);
+    // First in the parent ocmb, we need to look if it matches in the index
+    // given in the command Then we will go into the children of that ocmb for
+    // that specific target type
+    pdbg_for_each_class_target("ocmb", ocmb) {
+      // The target position didnt match ocmb fapi pos, so move on
+      if (getFapiUnitPos(ocmb) != i_target.pos)
+        continue;
+      if (pdbgClassString
```

Comment:
did not understand this why pdbgClassString empty should specify it is ocmb?

Reply:
// mapping table for ekb, ecmd and pdbg
const odyssey_chipUnit_t OdysseyChipUnitTable[] = {
    {ODYSSEY_NO_CU, "", ""},
    {ODYSSEY_MEMPORT_CHIPUNIT, "mp", "mem_port"},
    {ODYSSEY_PERV_CHIPUNIT, "perv", "ody_chiplet"},
    {ODYSSEY_OMI_CHIPUNIT, "omi", "omi"},
};

This is mapping table and when there is no chipunit, I have kept it empty. May be better to go with ocmb as the pdbg name. I will address that in the JIRA ticket. https://jsw.ibm.com/browse/PFEBMC-1931 Updated a comment there. After changing this i would like to test. So will address later